src/databricks/labs/ucx/source_code/graph.py

```diff
@@ -276,6 +276,11 @@ def resolve_import(self, path_lookup: PathLookup, name: str) -> MaybeDependency:
         assert self._next_resolver is not None
         return self._next_resolver.resolve_import(path_lookup, name)
 
+    def resolve_library_pip(self, path_lookup: PathLookup, name: str) -> MaybeDependency:
```

Comment:
No, this goes against the rationale provided in the PR: introduces confusion and complexity